babel.config.js

```diff
@@ -1,3 +1,13 @@
+const path = require('path');
+const glob = require('glob');
+
+const mapper = {
```

Comment:
why do we need it?

Reply:
can't we use this mapper to be able to transform imports for `layouts`, `TooltipPosition` and other stuff that has different path?